playbooks/tests/unit_tests.yml

```diff
@@ -3,7 +3,16 @@
   hosts: localhost
   gather_facts: False
   tasks:
-          - name: Install Test Reqs
+          - name: Install yum Test Reqs
+            yum:
+                    name: "{{ packages }}"
+                    state: present
+            vars:
+                    packages:
+                          - gcc
+                          - python-devel
+            become: yes
```

Comment:
@14rcole I dont think you need sudo here if we are running as root user in the container